ansible/roles/deploy/tasks/main.yml

```diff
@@ -47,3 +38,6 @@
     group: 'root'
   notify:
     - 'start app'
+- name: 'Install valgrind'
+  apt:
+    name: valgrind
```

Comment:
I wasn't sure if this is the best place to add this.

Maybe the whole `deploy` task should be renamed now that the service binary is built on-host and this task is mostly setting up the systemd unit and config?

Reply:
I think installing valgrind here makes sense, by lack of a better place (or we could rename the `cpufreq` role to something else like `misc` and put dependencies like valgrind and git there).

If we keep valgrind here, I'd suggest making it the first step of the task, to ensure valgrind is installed before the app runs (it probably doesn't make a difference 99% of the time, but feels more correct and might save someone down the line a confusing error)

Regarding the name of the `deploy` roole, I'm having trouble thinking of something more suitable... Maybe `bench-app` would do. Or did you have another name in mind?

Reply:
> If we keep valgrind here, I'd suggest making it the first step of the task

Makes sense to me :+1: 

> Maybe bench-app would do. Or did you have another name in mind?

That seems like a good name to me :-)

Reply:
> If we keep valgrind here, I'd suggest making it the first step of the task

Fixed.

> Maybe bench-app would do. Or did you have another name in mind?

I went with `bench-app-service`.